src/NzbDrone.Core/Notifications/MediaBrowser/MediaBrowser.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentValidation.Results;
+using NzbDrone.Common;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Tv;
+
+namespace NzbDrone.Core.Notifications.MediaBrowser
+{
+    public class MediaBrowser : NotificationBase<MediaBrowserSettings>
```

Comment:
Any particular reason why? Currently we use the class name as the name when selection the type, so this would need to change or look bad, but I don't think re-factoring all that is warranted currently.